pkg/ipc/ipc.go

```diff
@@ -585,7 +585,7 @@ func makeCommand(pid int, bin []string, config *Config, inFile, outFile *os.File
 	if inFile != nil && outFile != nil {
 		cmd.ExtraFiles = []*os.File{inFile, outFile}
 	}
-	cmd.Env = []string{}
+	cmd.Env = os.Environ()
```

Comment:
I tried to track the reason for addition of this line, but it really goes back to the initial commit... let's assume this was just a proactive isolation measure.
But `os.Environ()` is the default:
https://golang.org/pkg/os/exec/#Cmd
it's usually not spelled explicitly (just like starting a command in shell).
So please drop this line instead.